content/participants/ttjc@KumarHemanth342/index.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Hemanth Kumar"
+type: "participant"
+name: "Hemanth Kumar"
+github-link: "https://github.com/kumar342"
+linkedin-link: "https://www.linkedin.com/in/hemanth-kumar-83759015a/"
+looking-for: "job"
+description: "Mongodb | Express | React | Node"
+---
+
+Hi! I am Hemanth Kumar and let me walk you through my intro, projects and blogs.
+
+### introduction
+
+I am a web developer and having skills like React and Node.
+
+## why are you taking this challenge?
+
+Facing the challenge is good to improve the skills or knowledge.
+
+## something interesting about you?
+
+I,m a friendly guy and with a good look.
+
+### help I need?
+
+It would be good if someone tells me how to do auth for apps.
+
+### how to contact me?
+
+- [LinkedIn] https://www.linkedin.com/in/hemanth-kumar-83759015a/
```

Comment:
Please use correct markdown format for link

Reply:
Hey kanishk,
I changed the markdown. Please review it .